workflows/upload.nf

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
It would be preferential to keep this file removed as CLIMB serves no purpose for us

Reply:
Yes, sure.